lib/libc/minimal/include/string.h

```diff
@@ -25,6 +25,7 @@ extern size_t strlen(const char *s);
 extern size_t strnlen(const char *s, size_t maxlen);
 extern int    strcmp(const char *s1, const char *s2);
 extern int    strncmp(const char *s1, const char *s2, size_t n);
+extern char  *strtok_r(char *str, const char *sep, char **sotre);
```

Comment:
s/sotre/store/ (or follow the [POSIX declaration](https://pubs.opengroup.org/onlinepubs/9699919799/functions/strtok.html) and make it `state`)

Reply:
Changed all store refs to `state`.